src/background/utils/moduleManager.ts

```diff
@@ -342,4 +361,19 @@ export default class ModuleManager {
 
     return null
   }
+
+  private async _deepReplaceByKey(callback: any, data: any, keyToReplace: string) {
```

Comment:
move _deepReplaceByKey to common/helpers and reuse in src/background/services/featureService.ts => uploadModules => addStylesToZip